pkgs/tools/security/pynitrokey/default.nix

```
@@ -4,47 +4,50 @@
 , nrfutil
 , libnitrokey
 , nix-update-script
+, installShellFiles
 }:
 
 with python3Packages;
 
 buildPythonApplication rec {
   pname = "pynitrokey";
-  version = "0.4.40";
+  version = "0.4.41";
   format = "pyproject";
 
   src = fetchPypi {
     inherit pname version;
-    hash = "sha256-Hu+8UooDzv4GhkWt0sCckQQyHjWn4V/zt2ADlVCoHmk=";
+    hash = "sha256-QszxzZ2I/Bc7stcUh02w6JW/UlF9l77ZyaeWP9TXhPY=";
   };
 
   propagatedBuildInputs = [
     certifi
     cffi
     click
-    click-aliases
     cryptography
     ecdsa
     frozendict
     fido2
     intelhex
     nkdfu
-    nrfutil
     python-dateutil
     pyusb
     requests
-    semver
     spsdk
     tqdm
-    urllib3
     tlv8
     typing-extensions
+    pyserial
+    protobuf
+    click-aliases
+    semver
+    nethsm
     importlib-metadata
```

You removed `click-aliases`, but it seems it is still in use: https://github.com/Nitrokey/pynitrokey/blob/v0.4.41/pyproject.toml#L41

```suggestion
    certifi
    cffi
    click
    click-aliases
    cryptography
    ecdsa
    fido2
    frozendict
    importlib-metadata
    intelhex
    nethsm
    nkdfu
    protobuf
    pyserial
    python-dateutil
    pyusb
    requests
    semver
    spsdk
    tlv8
    tqdm
    typing-extensions
```

---

I just re-ordered the dependencies to match the ordering in the pynitrokey/pyproject.toml so that it's easier to copy paste them. click-aliases is still there.